__tests__/database/schema/UserSchema.test.ts

```diff
@@ -495,7 +496,8 @@ describe('TESTING UserSchema', () => {
         countryCode: 'countryCode',
         createdAt: 'createdAt',
         lastActiveAt: 'lastActiveAt',
-        profilePicture: 'profilePicture',
+        profilePicture:
+          'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded URL for the default profile picture could potentially cause issues if the resource at that URL changes or becomes unavailable. Consider storing this default image locally or in a more controlled environment, and reference it from there.
```diff
-        profilePicture: 'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
+        profilePicture: '/path/to/local/default-profile-picture.png',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->